### ISSUE TITLE
[Design] Auth Profile: Bad Address Indicator: Review work and mock up indicator messages

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#35229]
## Background
VA Profile has an indicator in their data source that can notify us if an address has been flagged as bad because mail has been returned, or some other event that we have not yet determined.  This work is to mock up what a "bad address" message would look like in the Profile Contact Info UI if the mailing and/or home address were flagged as bad.

## Tasks
- [x] Meeting to discuss scope of this work
- [x] Wireframe possible solutions to display a bad address message on the main profile page and on the contact information page.
- [x] Links to mockups have been added in a comment on this ticket.
- [x] Present wireframes to team 
- [x] Present wireframes to design team for feedback
- [ ] Schedule design intent review

## Acceptance Criteria
- [x] Meeting to discuss scope of this work has been held
- [x] Wireframe possible solutions to display a bad address message on the main profile page and on the contact information page.
- [x] Links to mockups have been added in a comment on this ticket.
- [x] Present wireframes to team 
- [x] Present wireframes to design team for feedback
- [ ] Schedule design intent review

## How to configure this issue
- Add label for practice area (frontend, backend, design)
- Assign to a Profile team member

**Comment by cgednalske** [comment]
Quick update here! I took the very low-fi design exploration for bad address indicator ([Mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1645053264056/07893207e0e0b79d3a6d55b0c02541319282f19d?sender=u28f508d646c449cc1afe4873)) to Platform Design Office Hours and received some initial feedback. Now finishing up more polished low-mid fi wireframes with mobile as well and will be ready to meet with you all to go over them shortly.

**Platform Office Hours Feedback:**
They were pleased to see a low-fi version at these early stages, particularly to anticipate and address possible accessibility questions that could come up later.

Overall the direction is looking good and there is nothing that should block us, though they're going to look into their accessibility guidance to confirm if the alert at the top of Personal info page should be above or below the H1 Personal information heading. 

Other questions addressed:
- The additional alert at the top of Contact info is probably not necessary. Even with screen readers/screen magnification, the alert in mailing is close enough to the top that another alert at the top should not be needed and would be somewhat redundant. In addition, if using another alert at the top of Contact, we'd need to use an anchor link, which goes into the territory of altering design system components.

- When linking to the section, we should not automatically open the edit field (don't think we easily have this capability anyway, but we confirmed this would be unusual for screen reader users).

**Comment by cgednalske** [comment]
@MarciMcGuireGCIO - here are links to the updated Sketch files for [desktop](https://www.sketch.com/s/59857eb5-d9f9-4145-99d3-d9a1de2d0655/a/3OMD28m) and [mobile](https://www.sketch.com/s/59857eb5-d9f9-4145-99d3-d9a1de2d0655/a/bgnMrQ5) with shorter placeholder text!

**Comment by MarciMcGuireGCIO** [comment]
Thank you @cgednalske!  @tpharrison - have you heard anything back from Barry about whether we can clear a bad address flag without changing the address?  Thanx!

**Comment by MarciMcGuireGCIO** [comment]
@cgednalske are we ready to schedule a design intent review, or do we need to wait to find out about the question above?

**Comment by tpharrison** [comment]
@MarciMcGuireGCIO - I haven't heard anything back yet.  I'll try asking him on slack.

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @tpharrison !

**Comment by cgednalske** [comment]
@MarciMcGuireGCIO hmm...sounds like the answer to that question might change the design if it turns out we cannot and would need to add a specific flow to go to re-confirm the same address. So I'm not sure, but maybe we should wait on that?

**Comment by MarciMcGuireGCIO** [comment]
That's a really good point.  Thx, @cgednalske !  cc @Samara-Strauss

**Comment by Samara-Strauss** [comment]
Agree with Christina -- if you haven't already, you might want to open another ticket so you don't lose track of this.

**Comment by MarciMcGuireGCIO** [comment]
On it.  Thanks!

**Comment by tpharrison** [comment]
FYI - I reached out to Barry Wilbanks through Slack about an hour ago, but haven't heard anything back.

**Comment by MarciMcGuireGCIO** [comment]
New ticket for updating the UI to allow for validating an address that has been erroneously flagged invalid #38184

**Comment by Samara-Strauss** [comment]
> FYI - I reached out to Barry Wilbanks through Slack about an hour ago, but haven't heard anything back.

Can we keep these convos in public channels? That way, I can nudge or pull in other people if necessary. Also, it gives us greater capabilities to follow along as a team. I didn't see this in the VA Profile channel, so I am assuming this was via DM.

**Comment by tpharrison** [comment]
Yep - my bad.  I'll reach out to him in #va-profile.
